### PR TITLE
wip Add WORKERS_CI environment variable to wrangler interactivity check

### PR DESCRIPTION
**Pull request by cmsparks**
## What this PR solves / how to test

wip

Fixes [BANDA-406](https://jira.cfdata.org/browse/BANDA-406).

## Author has addressed the following

- Tests
  - [ ] TODO (before merge)
  - [ ] Tests included
  - [ ] Tests not necessary because:
- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
  - [ ] I don't know
  - [ ] Required
  - [ ] Not required because:
- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
  - [ ] TODO (before merge)
  - [ ] Changeset included
  - [ ] Changeset not necessary because:
- Public documentation
  - [ ] TODO (before merge)
  - [ ] Cloudflare docs PR(s): 
  - [ ] Documentation not necessary because:

### CODE REVIEW

**Review by jahands — APPROVED** [review]

File: packages/wrangler/src/is-interactive.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import { CI } from "./is-ci";
  * or you're piping values from / to another process, etc
  */
 export default function isInteractive(): boolean {
```
Comment (jahands) [inline packages/wrangler/src/is-interactive.ts:8]: many CLIs I've used respect CI=1 unless FORCE_COLOR=1 - I'm surprised we don't do that in wrangler

Comment (cmsparks) [reply packages/wrangler/src/is-interactive.ts:8]: The library that wrangler uses `is-ci` doesnt' seem to respect CI=1 unfortunately.

Comment (cmsparks) [reply packages/wrangler/src/is-interactive.ts:8]: We actually do, it just uses this external is-ci library (which imports the ci-info library): https://github.com/watson/ci-info/blob/master/index.js#L60

**Comment by penalosa** [comment]
@cmsparks is this ready to go?

**Comment by cmsparks** [comment]
Forgot about this PR, but we actually don't need it because we set `CI=1` in the build environment, which is checked by the `is-ci` package.

**Pull request closed without merging**